src/main/java/inventorysetups/InventorySetupsPersistentDataManager.java

```diff
@@ -108,14 +105,27 @@ public void updateConfig(boolean updateSetups, boolean updateSections)
 	{
 		if (updateSetups)
 		{
-			List<InventorySetupSerializable> issList = new ArrayList<>();
+			// Rather than escaping the name to pick a config key, instead use hash of the name so that changes to the escaping function or supported characters in the name string don't affect config keys. Also avoids issues from overly long names and prevents other plugins from using the config key as the source of truth for the name; encouraging the use of the name key within each setup's json.

```

Comment:
Put this comment on separate lines. Also, the second half is a bit unclear to me, can you elaborate?

Reply:
New phrasing should hopefully be clearer. I wanted to avoid representing the name twice on disk; in a real user-specified format (saved in the json under name key) and a potentially unstable escaped string; that would change if we change the escaping function. This way, plugins like bank tab layouts would definitely grab name from the json instead of using the escaped name.